### PR TITLE
Add highlight-line toggle to core

### PR DESCRIPTION
**Pull request by jessejanderson**
Needs testing as I'm not sure how to run this locally.

I think the keybindings should work but I'm not 100% sure.

### CODE REVIEW

File: src/cljs/proton/layers/core/keybindings.cljs [hunk 1]
```diff
@@ -162,6 +162,9 @@
              :n {:title "line numbers"
                  :target (fn [atom] (.getView (.-views atom) (.getActiveTextEditor (.-workspace atom))))
                  :action "editor:toggle-line-numbers"}
+             :h {:category "highlight"
+                 :h {:title "highlight current line"
```
Comment (dvcrn) [inline src/cljs/proton/layers/core/keybindings.cljs]: any particular reason why you put this into a separate category?

**Comment by dvcrn** [comment]
To run locally, just install leiningen and java on your local machine, go into the proton directory and hit `lein run -m build/dev`. Once that's done, delete proton-mode from your editor, close atom, got into the `plugin/` folder and run `apm link`

File: src/cljs/proton/layers/core/keybindings.cljs (hunk 1 above)
Comment (jessejanderson) [inline src/cljs/proton/layers/core/keybindings.cljs]: I was following the spacemacs keybinding - it has a few additional "highlight" toggles so `SPC t h h` is the keybinding for line highlighting.

Comment (dvcrn) [inline src/cljs/proton/layers/core/keybindings.cljs]: Hmm I think with only 1 highlighting item we shouldn't make command chains too long. Do you think you could find a different keybinding under `t` you would be happy with?

Comment (jessejanderson) [inline src/cljs/proton/layers/core/keybindings.cljs]: well just having it under a single `h` would probably be fine and mostly match `spacemacs` binding. I wasn't sure how closely you were wanting to mimic it.

Comment (dvcrn) [inline src/cljs/proton/layers/core/keybindings.cljs]: All right - let's leave it on highlights then. We can always change it later. 
Though you could change the keybinding to `h l`? It's highlighting the current line and not highlighting the highlight :smile:

**Comment by jessejanderson** [comment]
Updated the keybinding to `SPC t h l`.

**Pull request merged**